libqtile/backend/x11/window.py

```diff
@@ -2080,16 +2084,16 @@ def handle_ConfigureRequest(self, e):  # noqa: N802
         else:
             width, height, x, y = self.width, self.height, self.x, self.y
 
-        if self.group and self.group.screen:
-            self.place(
-                x,
-                y,
-                width,
-                height,
-                self.borderwidth,
-                self.bordercolor,
-            )
+        self.place(
```

Comment:
Couldn't this move windows between groups as you're now ignoring whether or not the window is in a visible group? That can't be desirable behaviour/